CODE_REVIEW.md

```diff
@@ -0,0 +1,36 @@
+# Are there any problems or code smells in the app?
+
+- __Resolved__: `book-search.component.html` => We should replace format date function as native date pipe because format date function is called when change detection cycle is triggered where as native date pipe is triggered when input data is modified.
+- `book-search.component.html` => Responsive design is not handled for list of books.
+- __Resolved__: `book-search.component.ts` => Memory leak: Unsubscribe missing for subscription done on getAllBooks. Using async pipe Instead of subscribe will allow to access to data in the template without creating extra property. Angular automatically unsubscribes from the observable on component destroy. This prevents the problem of orphan subscriptions and memory leak, also we don't need to hold the subscription with extra property in the component.When we use async pipe here ,unsubscription done when component destroy.
```

Comment:
Take care of proper casing of word, punctuation marks. Use any editor to correct the sentence.